src/Food.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import Modal from './Modal';
+import { deleteFood } from './services/calories'
+import { fetchRecipes } from './services/recipes'
+
+class Food extends Component {
+  constructor() {
+    super()
+    this.state = {
+      showModal: false,
+      hover: false,
+      recipes: null
+    }
+  }
+
+  getRecipes = async () => {
+    const recipes = await fetchRecipes(this.props.data.name);
+    this.setState({
+      recipes: recipes
+    })
+    console.log(this.state.recipes)
+  }
+
+  deleteRecord = () => {
+    const userID = this.props.userID;
+    const mealID = this.props.mealID;
+    const foodID = this.props.data.id;
+    // deleteFood(userID, mealID, foodID);
+    console.log(userID, mealID, foodID);
+  }
+
+  showModal = () => {
```

Comment:
Peregrine and I had an offline discussion about making showModal and hideModal functions in the Food component, rather than in the App itself, alongside the other modal functions. There were conflicts with the other modal rendering by using the same show and hide, and rather that polluting App with functions for each different modal, it made sense organizationally to keep it in the Food component.